### PR TITLE
Get user reviews by email

### PR DESCRIPTION
**Pull request by madhura112**
Get user reviews by email
Pair Programmers: Parkaranjot Singh, Alexander Goebel, Madhura Gangal

### CODE REVIEW

File: backend/models/ratings_model.js [hunk 1]
```diff
@@ -20,8 +20,27 @@ const ratingsSchema = new Schema({
...
+    user_email: {
+        type: String,
+        required: true
+    },
+    user_name: {
+        type: String,
+        required: true
     }
 });
+// not needed
```
Comment (parkaransingh) [inline backend/models/ratings_model.js:33]: Get rid of this function in future commit

File: backend/server.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ import userRouter from './routes/users_route.js';
 
 app.use(cors());
 
-mongoose.connect('mongodb://127.0.0.1:27017/CivilityOptics', { useNewUrlParser: true });
-
+//mongoose.connect('mongodb://127.0.0.1:27017/CivilityOptics', { useNewUrlParser: true });
+mongoose.connect('mongodb+srv://CivilityOptics:Dontpush2134@civility1.7zzflo5.mongodb.net/?retryWrites=true&w=majority', { useNewUrlParser: true });
```
Comment (parkaransingh) [inline backend/server.js:20]: for future commits lets put this in the env file

**Review by parkaransingh — APPROVED** [review]
Looks good to go to main now.

**Review by Spellbreeze — APPROVED** [review]
Remove ratings.Schema.statics.findByEmail if unnecessary

**Pull request merged**